share/completions/exif.fish

```diff
@@ -0,0 +1,76 @@
+function __get_target_files_tags
+  for target in (string match -v -- '-*' (commandline -po)[2..])
+    string replace -f '*' '' (exif --list-tags "$target" 2> /dev/null)[2..] | string replace -r '(\s+-){4}' '' | string split -m1 ' '  | string trim
+  end
+end
+
+function __get_potential_targets
+  set -l matching_files (commandline -t)*
+  for file in $matching_files
+    if test -d "$file"
+      echo "$file/"
+    else if exif "$file" &> /dev/null
```

Comment:
(You don't need to quote $file here.)

Potentially spawning an exif process for each file in the current directory sounds a bit wild.
Disabling file completion makes us miss out on some nice features (handling of $variables, ~, `a/b/c` expansion).
I don't know exif but it should probably just complete normal files, or perhaps use `__fish_complete_suffix` to prefer some common suffixes.